application/utils.py

```
@@ -53,6 +59,9 @@ def send_email(to_email, subject, body):
     server_ssl.login(sender_email, sender_password)
 
     server_ssl.sendmail(sender_email, to_email, msg.as_string())
+
+    logger.info(f'Successfully sent email from {sender_email} to {to_email}')
```

Ruff

Logging statement uses f-string

(G004)